Source/Core/DolphinQt/Config/Mapping/MappingIndicator.cpp

```diff
@@ -356,8 +356,14 @@ void TiltIndicator::Draw()
 {
   WiimoteEmu::EmulateTilt(&m_motion_state, &m_group, 1.f / INDICATOR_UPDATE_FREQ);
 
-  const auto adj_coord =
-      Common::DVec2{-m_motion_state.angle.y, m_motion_state.angle.x} / MathUtil::PI;
+  auto adj_coord = Common::DVec2{-m_motion_state.angle.y, m_motion_state.angle.x} / MathUtil::PI;
+
+  // Angle may extend beyond 180 degrees when wrapping around.
+  // Apply modulo to draw within the indicator.
+  // Scale down the value a bit so +1 does not become -1.
+  adj_coord /= 1.0001f;
+  adj_coord.x = std::fmod(adj_coord.x + 3, 2) - 1;
```

Comment:
Why +3 and not +1? Actually, could you explain what the constants are?

Reply:
Values may be as low as -2 when the wrap-around logic runs. Only adding one wouldn't bring this into the (-1,+1) range for proper display.
I can try to add a comment or make it more clear somehow.